app/dts/behaviors/mod_tap.dtsi

```diff
@@ -0,0 +1,9 @@
+/ {
+	behaviors {
+		mt: behavior_mod_tap {
+			compatible = "zmk,behavior-mod-tap", "zmk,behavior-global";
```

Comment:
Mod tap is both, a keymap bound and global bound behavior. This is because if there is *some other* keycode pressed while it is held, then it avoids sending the bound keycode, and is used just as a modifier. We can expect other behaviors to act like this too, like layer-tap.